software_reseller/data/product_template.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <data noupdate="1">
+        <record id="software_reseller.product_template_1_f06ecb8a" model="product.template" context="{'create_product_product': False}">
+            <field name="service_to_purchase" eval="True"/>
+            <field name="purchase_method">purchase</field>
```

Comment:
No need of fields:
- "purchase_method" => Should be set on the "default_purchase_method" on the res_config_settings, to do so you need to correctly set the ir_default before creating the products. => or we don't enforce it in this module and remove the field to use what the user has in his database
- "invoice_policy" => default value
- "type" => computed based on the detailed_type

Reply:
same for all products